src/middleware/authentication.ts

```diff
@@ -149,18 +151,23 @@ export class Authentication {
 							error: `Looks like user with logToId ${_resp.data.userId} is not assigned to any CredentialService customer`,
 						});
 					}
-					response.locals.customer = user.customer;
-					response.locals.user = user;
-				} else if (!(['/account/create'].includes(request.path) && _resp.data.customerId == '')) {
-					// allow bootstrap apis to create a customer if there is not customerId
-					const customer = await CustomerService.instance.get(_resp.data.customerId);
+					customer = user.customer;
+				} 
+				if (_resp.data.customerId !== '' && !customer) {
+					customer = await CustomerService.instance.get(_resp.data.customerId);
 					if (!customer) {
 						return response.status(StatusCodes.NOT_FOUND).json({
-							error: `Looks like user with logToId ${_resp.data.customerId} is not found`,
+							error: `Looks like customer with id ${_resp.data.customerId} is not found`,
 						});
 					}
-					response.locals.customer = customer;
 				}
+				if (!customer && !user) {
```

Comment:
This will throw an unauthorized error for /account/create api.

so we should include `!['/account/create'].includes(request.path) ` in this condition to avoid it